dev/release/02-source-test.rb

```diff
@@ -84,7 +84,12 @@ def test_csharp_git_commit_information
   def test_python_version
     source
     Dir.chdir("#{@tag_name_no_rc}/python") do
-      sh("python3", "setup.py", "sdist")
+      # Meson dist must be run from a VCS, so initiate a dummy repo
```

Comment:
Er, really? This seems a bit clumsy (and may annoy downstream packagers).

Reply:
Yes this looks like a requirement. See also our discussion in https://github.com/apache/arrow/pull/45854/files#r2014837402